backend/app/app/core/config.py

```diff
@@ -21,8 +23,6 @@ def getenv_boolean(var_name, default_value=False):
 DOCS_BASE_URL = BASE_URL + DOCS_STR
 REDOC_BASE_URL = BASE_URL + REDOC_STR
 
-FAKE_LOGIN = getenv_boolean("FAKE_LOGIN", False)
```

Comment:
Seems that now we can remove `FAKE_LOGIN` variable from here:
https://github.com/WinnerOK/SWP_sport_back/blob/91c01726e9e8625b866179eeb651109b9a5d63c5/compose/docker-compose.yml#L30-L31